### ISSUE TITLE
Missing MWT tokens

### ISSUE THREAD

**Issue by kanayamah** [UniversalDependencies/UD_English-EWT#158]
The token representation `29-30  didn't` is missing here.
```
# sent_id = weblog-blogspot.com_gettingpolitical_20030906235000_ENG_20030906_235000-0002
# text = He could be killed years ago and the israelians have all the reasons, since he founded and he is the spiritual leader of Hamas, but they didn't.
27	but	but	CCONJ	CC	_	29	cc	29:cc	_
28	they	they	PRON	PRP	Case=Nom|Number=Plur|Person=3|PronType=Prs	29	nsubj	29:nsubj	_
29	did	do	VERB	VBD	Mood=Ind|Tense=Past|VerbForm=Fin	4	conj	4:conj:but	SpaceAfter=No
30	n't	not	PART	RB	_	29	advmod	29:advmod	SpaceAfter=No
31	.	.	PUNCT	.	_	4	punct	4:punct	_
```

In the current `dev` branch, I found 108 cases similar to #145.  Here is the list of the cases, with sentence IDs.  In all of these cases, there is no whitespace before or after the MWT (***e.g.*** period in the above case) thus I guess the converter from v2.6 overlooked them.  It is be critical to train and evaluate English tokenization.

**dev** 
```
didn't	weblog-blogspot.com_gettingpolitical_20030906235000_ENG_20030906_235000-0002
Don't	weblog-blogspot.com_marketview_20050210075500_ENG_20050210_075500-0003
Don't	weblog-blogspot.com_marketview_20050210075500_ENG_20050210_075500-0003
Arafat's	weblog-juancole.com_juancole_20041120060600_ENG_20041120_060600-0003
It's	newsgroup-groups.google.com_hiddennook_edef226e24a57863_ENG_20051116_085200-0005
We're	newsgroup-groups.google.com_hiddennook_5380fdd00f8e5e56_ENG_20050926_194800-0013
McDonalds	answers-20090625143116AAhML1y_ans-0001
can't	answers-20111107180248AAnQ3aE_ans-0002
wasn't	answers-20111108083304AAryBu9_ans-0006
I'm	answers-20100416184011AAtazxe_ans-0002
cant	answers-20090205181308AAZghOH_ans-0009
they're	answers-20111108024148AAO8oFI_ans-0009
won't	reviews-294812-0002
dont	reviews-158740-0003
```
**test**
```
Wei's	newsgroup-groups.google.com_fineart_0339fc0ed4e53c5a_ENG_20050930_025500-0008
I'm	newsgroup-groups.google.com_jokecity_0566f0ba3b5f748f_ENG_20051125_240500-0007
isn't	newsgroup-groups.google.com_jokecity_0566f0ba3b5f748f_ENG_20051125_240500-0009
NASA's	newsgroup-groups.google.com_hiddennook_88969236563fa748_ENG_20050215_173600-0007
thats	answers-20111108084149AAbQBhq_ans-0003
I'm	answers-20111108102612AASRNao_ans-0002
Donalds	answers-20111107035344AAdi9dS_ans-0002
don't	answers-20081218053636AA9vV0u_ans-0006
don't	answers-20111108081911AAy6QeW_ans-0003
that's	answers-20110320195750AAkPbFG_ans-0006
I'm	answers-20111108050147AAOkFgL_ans-0002
```

**train**
```
Kerry’s	weblog-blogspot.com_alaindewitt_20040929103700_ENG_20040929_103700-0071
can't	weblog-blogspot.com_rigorousintuition_20050518101500_ENG_20050518_101500-0015
didn't	weblog-juancole.com_juancole_20040823064025_ENG_20040823_064025-0087
It's	weblog-blogspot.com_rigorousintuition_20060511134300_ENG_20060511_134300-0015
There's	weblog-blogspot.com_rigorousintuition_20060511134300_ENG_20060511_134300-0021
don't	weblog-blogspot.com_rigorousintuition_20060511134300_ENG_20060511_134300-0052
There's	weblog-blogspot.com_rigorousintuition_20060511134300_ENG_20060511_134300-0071
Bout's	weblog-blogspot.com_rigorousintuition_20060511134300_ENG_20060511_134300-0142
Don't	weblog-blogspot.com_rigorousintuition_20060511134300_ENG_20060511_134300-0233
haven't	email-enronsent26_01-0028
haven't	email-enronsent26_01-0040
Gotta	email-enronsent15_01-0038
it's	email-enronsent34_01-0068
I've	email-enronsent06_01-0022
its	email-enronsent10_01-0020
there's	email-enronsent14_01-0105
don't	email-enronsent05_02-0096
They're	newsgroup-groups.google.com_IndiaNewsWindow_1405662573fa84ed_ENG_20050831_110400-0006
It's	newsgroup-groups.google.com_IndiaNewsWindow_1405662573fa84ed_ENG_20050831_110400-0010
You're	newsgroup-groups.google.com_IndiaNewsWindow_1405662573fa84ed_ENG_20050831_110400-0031
There's	newsgroup-groups.google.com_alt.animals_02c2d614bfbf6b20_ENG_20050223_232900-0032
I'm	newsgroup-groups.google.com_alt.animals_02c2d614bfbf6b20_ENG_20050223_232900-0040
I've	newsgroup-groups.google.com_alt.animals_434fe80fb3577e8e_ENG_20031011_200300-0030
Devil's	newsgroup-groups.google.com_alt.animals_434fe80fb3577e8e_ENG_20031011_200300-0057
children's	newsgroup-groups.google.com_alt.animals_0084bdc731bfc8d8_ENG_20040905_212000-0072
I'm	answers-20111108091950AAA0Xg5_ans-0006
im	answers-20111108083309AAg9jwT_ans-0002
wouldnt	answers-20111108102621AA3hPqj_ans-0006
I'm	answers-20111108061322AAQXY59_ans-0002
it's	answers-20111108061322AAQXY59_ans-0004
it's	answers-20111108110610AA4bcXX_ans-0014
jacks	answers-20111108085945AAgJhOG_ans-0012
wouldn't	answers-20111108105400AASqPIh_ans-0006
doesn't	answers-20111108103158AARnMLC_ans-0011
it's	answers-20111108111010AASEk0S_ans-0008
wasn't	answers-20111106144630AAadR8l_ans-0009
i'm	answers-20111108094927AA5NjHj_ans-0015
I've	answers-20111108084416AAoPgBv_ans-0019
it's	answers-20111108101024AANYJpw_ans-0023
don't	answers-20111108101850AAhNuvz_ans-0014
don't	answers-20111108101850AAhNuvz_ans-0015
can't	answers-20111108111107AAlrzok_ans-0011
Dont	answers-20111108064026AA86V9T_ans-0022
wasn't	answers-20111108102428AAMzXRG_ans-0018
I'm	answers-20111108103957AAcF3iZ_ans-0005
I've	answers-20111108105520AA73Axw_ans-0018
I've	answers-20111108091921AAaLK4e_ans-0014
didn't	answers-20111108091921AAaLK4e_ans-0027
CANNOT	answers-20111108104724AAuBUR7_ans-0016
Cannot	reviews-295491-0005
I'm	reviews-345455-0003
dont	reviews-215460-0004
don't	reviews-150192-0001
I'm	reviews-313825-0004
Fraiser's	reviews-059416-0002
I'm	reviews-317594-0003
I've	reviews-346960-0005
aren't	reviews-223912-0001
wasn't	reviews-223912-0003
isn't	reviews-191666-0002
I'm	reviews-276689-0004
elses	reviews-158335-0006
family's	reviews-077298-0003
it's	reviews-314974-0007
isn't	reviews-317846-0008
don't	reviews-255261-0010
i'm	reviews-159371-0003
We're	reviews-376503-0002
we're	reviews-376503-0002
don't	reviews-335225-0006
hadn't	reviews-006970-0009
dont	reviews-247226-0005
isn't	reviews-120335-0004
You've	reviews-034501-0007
it's	reviews-292841-0007
I'd	reviews-361545-0004
I'm	reviews-231203-0002
Cannot	reviews-187266-0013
elses	reviews-235423-0004
that's	reviews-086839-0008
I'm	reviews-086839-0008
I'm	reviews-086839-0013
```

**Comment by kanayamah** [comment]
@nschneid may I assign this to you?

**Comment by nschneid** [comment]
This is the same issue as #156.

**Comment by kanayamah** [comment]
@nschneid sorry for duplication!  I found the case above in a similar script with yours, thus please use this list for confirmation.

**Comment by nschneid** [comment]
These should all be fixed in 2b19a29. If you see issues let us know. Thanks
